tldw_Server_API/tests/Admin/test_admin_webhooks_service.py

```
@@ -0,0 +1,252 @@
+
+
+def _make_pool_mock() -> MagicMock:
+    """Return a mock DatabasePool with async helpers."""
+    pool = MagicMock()
+    pool.fetchone = AsyncMock()
+    pool.fetchall = AsyncMock()
+    pool.execute = AsyncMock()
+    return pool
+
+
+@pytest.fixture()
```

Ruff (0.15.7)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)